docs/web/docs/1-getting_started/1-getting_started.mdx

```diff
@@ -30,67 +44,76 @@ a scoped role.
 Refer to the [AWS documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html)
 for specific guidance on configuring credentials.
 
-### Set up the Github Repositories
+## Set up the Github Repositories
 
-To get started with IAMbic, we're going to create Github repositories repository to store IAMbic templates for importing your cloud resources.
+Github repositories repository are used to store IAMbic templates for importing your cloud resources.
```

Comment:
How about: `Github repositories are used to store IAMbic templates and change summaries for your cloud resources.`

Reply:
Sounds good! thank you.